core-bundle/contao/classes/Crawl.php

```diff
@@ -360,6 +367,43 @@ private function generateSubscribersWidget(array $subscriberNames): Widget
 		return $widget;
 	}
 
+	private function generateMaxDepthWidget(): Widget
+	{
+		$name = 'crawl_depth';
+
+		$widget = new SelectMenu();
+		$widget->id = $name;
+		$widget->name = $name;
+		$widget->label = $GLOBALS['TL_LANG']['tl_maintenance']['crawlDepth'][0];
+		$widget->setInputCallback($this->getInputCallback($name));
+
+		$options = array();
+		$default = System::getContainer()->getParameter('contao.backend.crawl.max_depth');
```

Comment:
What exactly can I configure here now? This is now configuring the default, not the `max_depth`. Did you mean to implement `for ($i = 3, $i <= $maxDepth....`?

Reply:
Yes, you configure the default `max_depth`, so the drop-down menu shows `5` instead of `3` if that is your default. But we might as well drop the setting and just use the drop-down instead. Would you prefer that?

Reply:
I think the naming is confusing. Because `max_depth` is not correct for what it's doing now. Imho we can remove it and just have a max until 10 - I think it would be better to store this in the session to remember what you last used than having a config for that.

Reply:
Removed in 235fe9c9126eccd9a06305708a4940e686fb970b.